web/.prettierrc

```diff
@@ -5,5 +5,5 @@
   "semi": false,
   "singleQuote": true,
   "tabWidth": 2,
-  "trailingComma": es5,
+  "trailingComma": "es5",
```

Comment:
Remove trailing comma here, as JSON doesn't support it.

Reply:
JSON does not, no, but unless you only format JSON, it'd still be relevant for JS/TS. But I can remove it!